php-homework/h.w1.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$i=1;
+$z=0;
+
+while($i<=50) {
+
+    $z=$i+$z;
+    $i=$i+2;
+    
+    echo "$z <br>";
+}
+
+?>
```

Comment:
please fix like below.
take attention to space, indent,.... 
follow this coding style guide https://www.php-fig.org/psr/psr-2/
```
$i = 1;
$z = 0;

while ($i <= 50) {

    $z = $i + $z;
    $i = $i + 2;

    echo "$z <br>";
}
```